internal/plugin/provider.go

```diff
@@ -201,6 +202,7 @@ func (p *AivenProvider) DataSources(context.Context) []func() datasource.DataSou
 			billinggroup.NewDataSource,
 			billinggrouplist.NewDataSource,
 			project.NewDataSource,
+			usergrouplist.NewDataSource,
```

Comment:
This resource doesn't use any experimental endpoints, so looks like we can make it public